src/components/grid/GridComponent.vue

```diff
@@ -216,6 +216,9 @@ export default Vue.extend({
     padding: 0 1px;
     position: relative;
   }
+  .row-hover:hover:not(.select-row) {
```

Comment:
I think `.row-hover:hover:not(.select-row) {` can be changed to `.row-hover:hover {`
Maybe some legacy code? I don't think `row-hover` can interact with `select-row`